src/constants/categories.tsx

```diff
@@ -4,19 +4,19 @@ import {
   FeedbackIcon,
   HomeIcon,
   type IconType,
-  LikedIcon,
+  CommunityIcon,
   SettingsIcon,
 } from '../icons/Icons';
 
-export const mainCats = ['Home', 'Liked', 'Events'] as const;
+export const mainCats = ['Home', 'Community', 'Events'] as const;
```

Comment:
Change this to "My Community". Based on this design below from the figma
<img width="387" alt="image" src="https://github.com/UTDNebula/jupiter/assets/76502130/71399355-cbb3-4d88-920a-9afafca431b8">